languageBase/en.json

```diff
@@ -1,25 +1,26 @@
 {
-    "Ovuli":"Ovuli",
-    "Please select your language":"Please select your language",
-    "How can we call you?": "How can we call you?",
-    "How Long is your is cycle?": "How Long is your is cycle?",
-    "When was your last period?": "When was your last period?",
-    "Do you know your cycle length?":"Do you know your cycle length?",
+    "Ovuli": "Ovuli",
+    "Pleaseselectyourlanguage": "Please select your language",
```

Comment:
convert every Object keys in snake case.
for e.g: `Pleaseselectyourlanguage` will be `please_select_your_language`.